packages/wonder-blocks-icon-button/src/components/icon-button-core.tsx

```diff
@@ -225,9 +226,12 @@ const _generateStyles = (
             /**
              * States
              *
-             * Defined in the following order: hover, focus, active.
+             * Defined in the following order: hover, active, focus.
+             *
+             * This is important as we want to give more priority to the
```

Comment:
How do we feel about increasing css specificity so we don't have to rely on the order of css declarations? Could we have specific styles for `:active:focus-visible` and `:hover:focus-visible`?

Reply:
I would prefer to avoid that at the DS level at least, as consumers would need to use the same selector (or a higher one) if for some reason they need to override styles (similar to the `":hover:not([aria-disabled=true])"` case). I think keeping it simple at the WB level would be beneficial for consumers so they can rely on more resilient styles and selectors from the start. This is something that we'll need to revisit for sure once we convert WB to CSS modules.